Library/Formula/dmd.rb

```diff
@@ -59,7 +59,7 @@ def install
     # linked into opt by the time this build runs:
     conf.write <<-EOS.undent
         [Environment]
-        DFLAGS=-I#{include}/d2 -L-L#{lib}
+        DFLAGS=-I#{include}/dlang/dmd -L-L#{lib}
```

Comment:
`head` ignores revisions because it isn't version-controlled; `devel` will be updated with the `revision`. It goes at the top of the formula in this case:
```ruby
desc "D programming language compiler for OS X"
homepage "https://dlang.org/"
revision 1

stable do
```